### PR TITLE
Cleanup

### LINKED ISSUE #70: Dockerize ts-relayer tests
Annoying to have to run start and stop bash scripts.

Want to be able to run `docker-compose up` and have a testing environment.

### LINKED ISSUE #71: Fix `cargo doc`
Documentation generation fails with:
```
error: document output filename collision
The bin `schema` in package `mesh-lockup v0.1.0 (/home/meow/Dev/CosmWasm/mesh-securme as the bin `schema` in package `mesh-consumer v0.1.0 (/home/meow/Dev/CosmWasm/meOnly one may be documented at once since they output to the same path.
```

### LINKED ISSUE #72: Incorporate schema and ts codegen tools and checks from DAO DAO
Would be nice to have a better setup for this.

### PR DESCRIPTION
**Pull request by JakeHartnell**
Prepping for future work with a bunch of small clean up, mostly making the dev environment a bit nicer. Borrowed heavily from the [DAO DAO contracts repo](https://github.com/DA0-DA0/dao-contracts).

- Closes #72 
- Closes #70 
- Closes #71

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -139,15 +139,15 @@ jobs:
           path: ~/ibc
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package-lock.json" }}
```
Comment (Art3miX) [inline .circleci/config.yml:142]: I guess v1 just carried over from DAO DAO? would remove v1 from the whole file if its not needed here, or else might cause confusion (of where is v2?)

File: contracts/mesh-consumer/Cargo.toml [hunk 2]
```diff
@@ -41,3 +41,7 @@ thiserror        = { workspace = true }
 [dev-dependencies]
 mesh-testing  = { workspace = true }
 cw-multi-test = { workspace = true }
+
+[[bin]]
```
Comment (Art3miX) [inline contracts/mesh-consumer/Cargo.toml:45]: any link to docs about this? just to learn what it means for myself.

**Review by Art3miX — APPROVED** [review]
Just 2 small questions
LGM.

File: contracts/mesh-consumer/Cargo.toml (hunk 2 above)
Comment (Art3miX) [reply contracts/mesh-consumer/Cargo.toml:45]: nevermind, found it: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries

File: .circleci/config.yml (hunk 1 above)
Comment (JakeHartnell) [reply .circleci/config.yml:142]: > I guess v1 just carried over from DAO DAO? would remove v1 from the whole file if its not needed here, or else might cause confusion (of where is v2?)

This is actually not from DAO DAO! We use GitHub CI which I actually prefer.

**Pull request merged**